### ISSUE TITLE
Accessibility basics

### ISSUE THREAD

**Issue by AutoSponge** [thesysdev/crayon#319]
**Describe the bug**
The playground demo (https://www.openui.com/playground) for Login Form is coded incorrectly for accessibility. It's unclear from the demo if the failure is the component, the lang, or the translation.

**To Reproduce**
1. Go to https://www.openui.com/playground
2. Click on Login Form
3. Focus any input
4. Open devtools, go to the Accessibility pane, see that the accessible name is derived from placeholder rather than the visible label.

**Expected behavior**
Demos should showcase accessible code. Especially simple setups like a login form.

**Screenshots**
N/A

**Desktop (please complete the following information):**
 - OS: Any
 - Browser Any
 - Version Any

**Smartphone (please complete the following information):**
 - Device: Any
 - OS: Any
 - Browser Any
 - Version Any

**Additional context**
I'm an expert in accessibility. If contributors/maintainers want this pattern to get adopted beyond vibe coders, it needs to produce accessible markup.

It's also not clear from the lang spec whether or not standard props (e.g., aria-label) will get passed through if they're not explicit in the schema.
